server/src/main/java/io/littlehorse/server/streams/taskqueue/OneTaskQueue.java

```diff
@@ -79,17 +83,16 @@ public void onRequestDisconnected(PollTaskRequestObserver disconnectedObserver)
      *                        that was just
      *                        scheduled.
      */
-    public void onTaskScheduled(ScheduledTaskModel scheduledTaskId) {
+    public boolean onTaskScheduled(ScheduledTaskModel scheduledTaskId) {
         // There's two cases here:
         // 1. There are clients waiting for requests, in which case we know that
         // the pendingTaskIds queue/list must be empty.
         // 2. There are no clients waiting for requests. In this case, we just
         // add the task id to the taskid list.
-
         log.trace(
                 "Instance {}: Task scheduled for wfRun {}, queue is empty? {}",
                 hostName,
-                LHLibUtil.getWfRunId(scheduledTaskId.getSource().toProto().build()),
```

Comment:
Suggestion: get rid of `LHUtil.getWfRunId()`, and change `LHLibUtil()` to take in `TaskRunSourceOrBuilder`.

Reply:
Done. 
However, I believe that we will transform the model into a 'proto' just because LHUtil only expects a 'proto'.